cypress.json

```diff
@@ -3,5 +3,6 @@
     "testFiles": "**/*.feature",
     "postLogoutUrl": "https://www.gov.uk/done/lasting-power-of-attorney",
     "rootRedirectUrl": "https://www.gov.uk/power-of-attorney/make-lasting-power",
-    "numberOfGuidanceHelpTopics": 22
+    "numberOfGuidanceHelpTopics": 22,
+    "defaultCommandTimeout": 20000
```

Comment:
I'd like to remove the defaultCommandTimeout setting and leave it as is: all this seems to achieve is slowing down the time it takes for tests to fail, rather than helping them pass.

Reply:
Fair enough. done :)